lib/maintainers.nix

```diff
@@ -328,6 +328,7 @@
   schmitthenner = "Fabian Schmitthenner <development@schmitthenner.eu>";
   schristo = "Scott Christopher <schristopher@konputa.com>";
   scolobb = "Sergiu Ivanov <sivanov@colimite.fr>";
+  sebastianjordan = "Sebastian Jordan <sebastian.jordan.mail@googlemail.com>";
```

Comment:
the left hand side should be your github handle. the idea is to make it easier to contact you when an issue is opened against the package.